bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -29,34 +31,9 @@ export default class ActionsBar extends Component {
           <EmojiContainer />
         </div>
         <div className={styles.hidden}>
-          <ActionsDropdown />
-        </div>
-      </div>
-    );
-  }
-
-  renderForUser() {
-    return (
-      <div className={styles.actionsbar}>
-        <div className={styles.center}>
-          <MuteAudioContainer />
-          <JoinAudioOptionsContainer
-            handleJoinAudio={this.props.handleOpenJoinAudio}
-            handleCloseAudio={this.props.handleExitAudio}
-
-          />
-          {/*<JoinVideo />*/}
-          <EmojiContainer />
+          <ActionsDropdown {...{isUserPresenter}}/>
```

Comment:
That doesn't seem like a good practice, you should fix the centralization issue with css only. What you could do to achieve the centralization you desire is: make the `center` div absolute and have it's width be the entire width of the actions bar.